### DISCUSSION TITLE
use_column_names in redshift.copy()

### DISCUSSION THREAD

**Discussion by aeeladawy** [aws/aws-sdk-pandas#1438] [Ideas]
Is there a specific reason why use_column_names is not included in redshift.copy?

I created this draft PR to give it a shot. [Add use_column_names to redshift.copy()](https://github.com/awslabs/aws-data-wrangler/pull/1437)

Over in: https://docs.aws.amazon.com/redshift/latest/dg/copy-parameters-column-mapping.html

> You can specify a comma-separated list of column names to load source data fields into specific target columns. The columns can be in any order in the COPY statement, but when loading from flat files, such as in an Amazon S3 bucket, their order must match the order of the source data.

Does that mean the output of the parquet files cannot be reordered for the COPY statement?

**Comment by kukushking** [comment]
No particular reason, thanks, PR adding this is very welcome!
